share/availability/full/reconstruction_test.go

```diff
@@ -181,35 +181,15 @@ func TestShareAvailable_DisconnectedFullNodes(t *testing.T) {
 	light.DefaultSampleAmount = 20 // s
 	const (
 		origSquareSize = 16 // k
-		lightNodes     = 60 // c - total number of nodes on two subnetworks
```

Comment:
Did you look at git blame? I bet it was a debugging value accidentally committed.

Reply:
This was the original value committed

Reply:
Ok, could you also double values for other reconstruction tests?

Reply:
I checked the git blame, and it shows that it was committed with this value because in the ec3793fc I moved files over, but this look like a fully new file when it's not 😒. I didn't look for the original one.